packages/core/test/financial-templates/perpetual-multiparty/PerpetualPositionManager.js

```diff
@@ -1051,76 +1074,75 @@ contract("PerpetualPositionManager", function(accounts) {
 
     assert.equal((await positionManager.fundingRate()).cumulativeMultiplier.toString(), toWei("1.05"));
 
-    // Set the funding rate to a negative funding rate of -0.00001 in the store and apply it for 10k seconds. New funding rate
-    // should be 1.05 * (1 - -0.00001 * 10k) = 0.945
-    await setFundingRateAndAdvanceTime(toWei("-0.00001"));
+    // Set the funding rate to a negative funding rate of -0.000001 in the store and apply it for 10k seconds. New funding rate
```

Comment:
The following changes came about originally because I set a bound on the funding rate on Perpetual construction, but now that the bound itself can be configured, to no bound for example, the changes are not technically neccessary. However, I'm going to keep them because they are more readable than before.